ntoskrnl/ps/process.c

```diff
@@ -1614,4 +1677,234 @@ NtOpenProcess(OUT PHANDLE ProcessHandle,
     /* Return status */
     return Status;
 }
+
+PEPROCESS NtpGetAdjacentProcess(const BOOLEAN is_backwards, const PEPROCESS current)
```

Comment:
Please use consistent code style in the kernel ("MyVariable" instead of "my_variable", etc...)